backend/src/controllers/review.controller.ts

```diff
@@ -122,6 +126,19 @@ The customer may also provide a list of previous replies that did not satisfy hi
     }
   }
 
+  async uploadViaReviewsReceiver(req: AuthRequest<{}, IReview>, res: Response) {
```

Comment:
The [swagger you created for the path](#diff-c4e2f08d7622c5006dfa082af66bc53ef599eec8167889a441c68a91189ad78fR318) specifies `req.body` is an array of IReview (`IReview[]`)
This function specifies just `IReview`